package.json

```diff
@@ -108,5 +108,8 @@
     "fail": [
       "@semantic-release/github"
     ]
+  },
+  "resolutions": {
+    "react-helmet/react-side-effect": "^2.1.0"
```

Comment:
No le ponés un rango de versiones?

Reply:
Pero funciona eso? te entendi que me pasaste algo en beta O_o

Reply:
```suggestion
    "react-helmet/react-side-effect@>=1.0.0 <2.1.0": "react-helmet/react-side-effect@^2.1.0"
```

Reply:
I've tried this and I get a warning:
```
yarn install v1.17.3
warning Resolution field "react-helmet/react-side-effect@^2.1.0" has an invalid version entry and may be ignored
``` 
So I'll leave this as is.

Reply:
Did you try this?

```suggestion
    "react-helmet/react-side-effect@>=1.0.0 <2.1.0": "^2.1.0"
```

Reply:
Si no hacemos algo así, cuando `react-helmet/react-side-effect` se actualice a la versión 3, vamos a seguir con la `2.x.x`

Reply:
La otra opción es documentarnos mejor antes de hacer el cambio.

Reply:
Esto ultimo si que funciona, gracias <3

Reply:
Corri el test, recheckie el yarn lock y al final no funciona, sigo investigando. (en lo ultimo que subi se ve, en el lock)